plugin/trino-tpch/src/main/java/io/trino/plugin/tpch/TpchMetadata.java

```diff
@@ -356,6 +357,9 @@ private static ColumnStatistics toColumnStatistics(ColumnStatisticsData stats, T
         if (min.isEmpty() || max.isEmpty()) {
             return Optional.empty();
         }
+        if (columnType instanceof DecimalType) {
+            return Optional.empty();
```

Comment:
That's a weird change. The commit says "Support decimal columns in Tpch statistics provider", but it looks like removing support for it.

Reply:
Without this line it fails on `DecimalType` being not supported. Perhaps a better solution would be to try to return values, but that requires a bigger refactor. I need this change just to unblock running tpch queries with decimal columns enabled.

Reply:
> Perhaps a better solution would be to try to return values

agreed

> but that requires a bigger refactor.

i hoped not. The actual ranges should be the same, and `io.trino.spi.statistics.ColumnStatistics` is already "normalized" into doubles, so we should be able use the min/max values we have both for doubles and decimals.

Reply:
Hmm, you are probably right, let me take a closer look.

Reply:
Actually yeah, the stats for decimal columns are available and it was super easy to propagate them. Thanks for the suggestion. Updated.